### ISSUE TITLE
Upgrade To Manifest Version 3 

### ISSUE THREAD

**Issue by null93** [null93/milk-cookie-manager#34]
This will involve converting the background script to a service worker. Also some minor manifest changes need to happen as well.

**Comment by null93** [comment]
Plugin is ready to be converted to Manifest v3:

```
node tools/migrate-manifest.js static/manifest.json dist/manifest.json
```

Problem is no stable channel supports service workers. Will have to wait to convert manifest versions. Should be good to go for Chrome 88.

**Comment by null93** [comment]
Chrome 88 started accepting v3, Firefox still does not. Their timeline estimates some time early 2022. Holding off migrating to v3 until then since chrome still accepts v2.

source: https://blog.mozilla.org/addons/2021/05/27/manifest-v3-update/

**Comment by null93** [comment]
Fixed in 2.4.0
